setup.py

```diff
@@ -16,65 +16,52 @@
 from setuptools import setup, find_packages
 import os.path
 
+
 def read_file(*args):
     path = os.path.join(os.path.dirname(__file__), *args)
-    file_contents = open(path, 'r').read()
-    return file_contents + '\n\n'
-
-def alltests():
-    import os
-    import sys
-    import unittest
-    # use the zope.testrunner machinery to find all the
-    # test suites we've put under ourselves
-    import zope.testrunner.find
-    import zope.testrunner.options
-    here = os.path.abspath(os.path.join(os.path.dirname(__file__), 'src'))
-    args = sys.argv[:]
-    defaults = ["--test-path", here]
-    options = zope.testrunner.options.get_options(args, defaults)
-    suites = list(zope.testrunner.find.find_suites(options))
-    return unittest.TestSuite(suites)
+    with open(path, 'r') as f:
+        return f.read() + '\n\n'
 
 setup(
     name='zc.sourcefactory',
     version='1.1.0.dev0',
     author='Zope Corporation and Contributors',
     author_email='zope-dev@zope.org',
-    url='http://pypi.python.org/pypi/zc.sourcefactory',
-    keywords='zope zope3 vocabulary source factory',
-    description='An easy way to create custom Zope 3 sources.',
+    url='https://github.com/zopefoundation/zc.sourcefactory',
+    keywords='zope vocabulary source factory',
+    description='An easy way to create custom Zope sources.',
     license='ZPL 2.1',
     long_description=(
+        read_file('README.txt') +
```

Comment:
It's README.rst, isn't it?

Reply:
🤕

Reply:
Time for weekend!

Reply:
I was looking at that Travis failure for like 5 minutes until I realized what was going on.

Reply:
@mgedmin Sorry for wasting your time for such a silly mistake of mine.